DependencyInjection/Compiler/FormFactoryCompilerPass.php

```diff
@@ -41,6 +41,7 @@ public function process(ContainerBuilder $container)
 
         // get factories
         $original = $container->getDefinition('form.extension');
+        parent::process($container);
```

Comment:
this is replacing arguments of the 'form.extension' service (aka $original). is that intended? in symfony 3.3 i get type mismatches because object (iterator?) instead of an array.